Source/WebInspectorUI/UserInterface/Views/ScopeBar.js

```diff
@@ -148,26 +150,28 @@ WI.ScopeBar = class ScopeBar extends WI.NavigationItem
 
     _itemSelectionDidChange(event)
     {
-        var sender = event.target;
-        var item;
-
-        // An exclusive item was selected, unselect everything else.
-        if (sender.exclusive && sender.selected) {
-            for (var i = 0; i < this._items.length; ++i) {
-                item = this._items[i];
-                if (item !== sender)
-                    item.selected = false;
-            }
-        } else {
-            let replacesCurrentSelection = this._shouldGroupNonExclusiveItems || !event.data.extendSelection;
-            for (var i = 0; i < this._items.length; ++i) {
-                item = this._items[i];
-                if (item.exclusive && item !== sender && sender.selected)
-                    item.selected = false;
-                else if (sender.selected && replacesCurrentSelection && sender !== item)
-                    item.selected = false;
+        if (this._ignoreItemSelectedEvent)
+            return;
+
+        this._ignoreItemSelectedEvent = true;
+
+        const deselectOtherItems = () => {
```

Comment:
i dont think we need to make this into a function since there's only one `return`, so we can simplify it as such
```js
if (event.target.selected) {
    let extendSelection = (event.data.extendSelection && !event.target.exclusive) || event.target === this._multipleItem;

    for (let item of this._items) {
        if (item === event.target)
            continue;

        if (extendSelection && !item.exclusive)
            continue;

        item.selected = false;
    }
}
```